payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/LpmSerializerTest.kt

```diff
@@ -564,4 +563,31 @@ class LpmSerializerTest {
             )
         }
     }
+
+    @Test
+    fun `Test placeholder parses correctly`() {
+        val serializedString =
+            """
+            {
+                "type": "au_becs_debit",
+                "fields": [
+                  {
+                    "type": "placeholder",
+                    "for": "name"
+                  }
+                ]
+            }
+            """.trimIndent()
+
+        val result = lpmSerializer.deserialize(serializedString)
+        result.onFailure {
+            println(it.message)
+        }
+        assertThat(result.isSuccess).isTrue()
+        result.onSuccess {
+            assertThat(it.fields.first()).isEqualTo(
```

Comment:
We don't have anything ensuring this gets called! You can use a count down latch or any other mechanism, but we should assert we actually hit this assert